src/components/rating/rating.tsx

```diff
@@ -0,0 +1,183 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Method,
+  Prop
+} from "@stencil/core";
+import { IFormComponent } from "../common/interfaces";
+@Component({
+  shadow: false,
+  styleUrl: "rating.scss",
+  tag: "gx-rating"
+})
+export class Rating implements IFormComponent {
+  inputId: string;
+
+  starShape = <polygon points="50,0 15,95 100,35 0,35 85,95" />;
+
+  svgViewport = {
+    viewBox: "0 0 100 100"
```

Comment:
```suggestion
    private viewBox: "0 0 100 100"
```

Reply:
It allows setting the variables as private, not the properties of objects.